tests/load_test

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env ruby
+
+require 'find'
+require_relative '../lib/const'
+require_relative '../lib/color'
+
+PKG_DIR = '../packages'
+
+# Add >LOCAL< lib to LOAD_PATH
+$LOAD_PATH.unshift '../lib'
+
+Find.find(PKG_DIR) do |filename|
```

Comment:
Will `Dir.glob("#{PKG_DIR}/*.rb").each do |filename|` better?

Reply:
With suggested change:
```
$ time ruby load_test
...snip...
All load tests successful.

real    0m23.684s
user    0m10.799s
sys     0m0.855s
```
Without suggested change:
```
$ time ruby load_test
...snip...
All load tests successful.

real    0m24.570s
user    0m10.678s
sys     0m1.312s
```
Negligible performance improvement?

Reply:
That's a 2 second improvement!

Run it several times to get an average though...

Reply:
It's a 2 second improvement without the suggested change. :)  I did run it several times and it varies a few seconds each time so the suggested change is effectively negligible.

Reply:
Try again after removing the following lines?
```
require 'find'
  if File.extname(filename) == '.rb'
```

Reply:
Yes, now I see what you are getting at.  Good observation!

Reply:
Definitely an improvement!
```
$ time ruby load_test
...snip...
All load tests successful.                                                                                                                                                                    

real    0m8.245s
user    0m1.082s
sys     0m0.245s
```

Reply:
I made the same change in the cycle_test script and that was an improvement also!